app/furniture/marketplace/order.rb

```diff
@@ -52,5 +52,135 @@ def marketplace_name
     def price_total
       [product_total, delivery_fee, tax_total].compact.sum
     end
+
+    # TODO: Add `square_environment` attribute to database/model
+    def square_client
+      @square_client ||= Square::Client.new(access_token: marketplace.square_access_token, environment: marketplace.square_environment)
```

Comment:
Since we're reading data from the `marketplace` to create the `square_client`, that implies to me that it maybe wants to live on the `marketplace`, and the `Order` class can `delegate :square_client, to: :marketplace`?